md/multi-hop-locks.md

```diff
@@ -124,6 +124,42 @@ An adversary may not choose the latest update to reply to but instead selects a
 This is not vulnerable to an attack similar to the [late message Wagner's attack](https://medium.com/blockstream/insecure-shortcuts-in-musig-2ad0d38a97da) because the adversary should not be able to trick the victim into signing a transaction with a different nonce.
 Instead, the victim's nonce is tied to a specific transaction which prevents the attacker from choosing a message for signing after seeing the victim's nonce.
 
+Cancellable payments
```

Comment:
The payments can't really be cancelled (you can't cancel an HTLC one it's extended, you either need cooperation or for the time-lock to expire), so IMO a better phrase is: two-phase payments. This solves the issues of the client saying the payment didn't arrive, to get the sender to pay again, which allows them to double settle.

Reply:
True. The paragraph does explain that though.